pkg/packaging/packaging.go

```diff
@@ -602,9 +602,9 @@ fi`
 func (p *PackageOptions) setupDirectories() error {
 	switch p.target.Platform {
 	case Linux, Darwin:
-		p.binDir = filepath.Join("/usr/local", p.Identifier, "bin")
+		p.binDir = filepath.Join("/opt", p.Identifier, "bin")
```

Comment:
I feel like I want to sleep on this one. It's not so clearly correct to me. `/usr/local` is host specific data, why do you think that's wrong?

Reply:
it's not host-specific. `/etc/...` contains host-specific config. `/usr/local` is for stuff _outside_ the packaging world. see also other comment in this pr with quotes and references to official packaging guides.

Reply:
These are vendor or site package, not things in the underlying package repositories. 

My read on these docs, is that an official distribution package must not use `/usr/local` because it's reserved for host, site, and third part software. Generally I think of launcher as fitting into that. 

If the goal were to get packages into the upstream distributions, then I'd completely agree. However, there is a small amount of additional work to make that feasible. This set of packaging is rooted in some ideas that wouldn't translate to being upstreamed.

All that said, I'm not sure I feel strongly about it. So I'm weighing re-training finger macros and updating docs, against making things feel better to a couple of people.

Reply:
from the [debian packaging docs](https://www.debian.org/doc/manuals/maint-guide/modify.en.html):

> On Debian this is reserved for private use by the system administrator

this means _local_ use, so stuff _outside_ packages.

this really means packages should _never_ put stuff in `/usr/local`, no matter who built the package.